assets/js/src/core/modules/element/dynamic-types/definitions/objects/data-related/components/many-to-many-relation/many-to-many-relation.tsx

```diff
@@ -79,7 +81,7 @@ export const ManyToManyRelation = (props: ManyToManyRelationProps): React.JSX.El
     }
   }, [JSON.stringify(props.value)])
 
-  if (props.isLoading === true) {
+  if (isLoading || props.isLoading === true) {
```

Comment:
The loading state here seemed to be a good idea, but it's a bit annoying that the full data type has a loading state - especially when a new item is added via the tree. The grid data type itself also provides a loading prop. Maybe it's better to just set the grid itself to loading instead of everything? Not sure how easy it's possible but in a perfect world we would see the loading spinner directly on row level for newly added items (and only for the initial load we use the global one).